defaults/main.yml

```diff
@@ -1,2 +1,2 @@
 ---
-terraform_version: "0.6.12"
+terraform_version: "0.6.12"
```

Comment:
fixed in [79c51f4d](https://github.com/azavea/ansible-terraform/pull/1/commits/79c51f4dfded92ee6e41546439d1e88ae93f418c)